src/lib/hooks/hasConnectionChange.js

```diff
@@ -22,21 +23,14 @@ export const useConnection = () => {
   return isConnection
```

Comment:
on useConnection:

1. `NetInfo.isConnected.fetch().then` should be called inside useEffect
2. useEffect should return cancel effect removing event listener

Reply:
fixed